foregroundServiceType 0x00000001 is not a subset of foregroundServiceType attribute 0x00000800 in service element of manifest file

Since upgrading to the latest version the app is frequently crashing. I have discovered it is when I have any options enabled on the "Send device information to server" page. The error in the logs is:

```
02-03 12:16:14.169  6383  6383 E AndroidRuntime: FATAL EXCEPTION: main
02-03 12:16:14.169  6383  6383 E AndroidRuntime: Process: org.openhab.habdroid, PID: 6383
02-03 12:16:14.169  6383  6383 E AndroidRuntime: java.lang.IllegalArgumentException: foregroundServiceType 0x00000001 is not a subset of foregroundServiceType attribute 0x00000800 in service element of manifest file
02-03 12:16:14.169  6383  6383 E AndroidRuntime: 	at android.os.Parcel.createExceptionOrNull(Parcel.java:3073)
02-03 12:16:14.169  6383  6383 E AndroidRuntime: 	at android.os.Parcel.createException(Parcel.java:3053)
02-03 12:16:14.169  6383  6383 E AndroidRuntime: 	at android.os.Parcel.readException(Parcel.java:3036)
02-03 12:16:14.169  6383  6383 E AndroidRuntime: 	at android.os.Parcel.readException(Parcel.java:2978)
02-03 12:16:14.169  6383  6383 E AndroidRuntime: 	at android.app.IActivityManager$Stub$Proxy.setServiceForeground(IActivityManager.java:7248)
02-03 12:16:14.169  6383  6383 E AndroidRuntime: 	at android.app.Service.startForeground(Service.java:862)
02-03 12:16:14.169  6383  6383 E AndroidRuntime: 	at androidx.work.impl.foreground.SystemForegroundService$Api31Impl.startForeground(SourceFile:193)
02-03 12:16:14.169  6383  6383 E AndroidRuntime: 	at androidx.work.impl.foreground.SystemForegroundService$1.run(SourceFile:129)
02-03 12:16:14.169  6383  6383 E AndroidRuntime: 	at android.os.Handler.handleCallback(Handler.java:958)
02-03 12:16:14.169  6383  6383 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:99)
02-03 12:16:14.169  6383  6383 E AndroidRuntime: 	at android.os.Looper.loopOnce(Looper.java:230)
02-03 12:16:14.169  6383  6383 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:319)
02-03 12:16:14.169  6383  6383 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:9063)
02-03 12:16:14.169  6383  6383 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
02-03 12:16:14.169  6383  6383 E AndroidRuntime: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:588)
02-03 12:16:14.169  6383  6383 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1103)
02-03 12:16:14.169  6383  6383 E AndroidRuntime: Caused by: android.os.RemoteException: Remote stack trace:
02-03 12:16:14.169  6383  6383 E AndroidRuntime: 	at com.android.server.am.ActiveServices.setServiceForegroundInnerLocked(ActiveServices.java:2269)
02-03 12:16:14.169  6383  6383 E AndroidRuntime: 	at com.android.server.am.ActiveServices.setServiceForegroundLocked(ActiveServices.java:1883)
02-03 12:16:14.169  6383  6383 E AndroidRuntime: 	at com.android.server.am.ActivityManagerService.setServiceForeground(ActivityManagerService.java:15992)
02-03 12:16:14.169  6383  6383 E AndroidRuntime: 	at android.app.IActivityManager$Stub.onTransact$setServiceForeground$(IActivityManager.java:12028)
02-03 12:16:14.169  6383  6383 E AndroidRuntime: 	at android.app.IActivityManager$Stub.onTransact(IActivityManager.java:3540)
```

If I turn all the "Send Device Information" options off it doesn't happen. If I toggle one of them on, it immediately crashes and then continues to crash randomly throughout the day.

### Environment data
#### Client
* Android version: 14
* Device model: Samsing SM-F946B
* App version: 3.17.1
* Build flavor Full
* Device language: English 

#### Server
* Server version: 4.3.2
* Reverse Proxy: 
* Authentication method None

#### openHAB Cloud
* Self hosted cloud or myopenhab.org: myopenhab 
* Authentication method: user/pass

### Logs
#### App log
<details>
  <summary>Click to expand</summary>

```

[applog.TXT](https://github.com/user-attachments/files/18658740/applog.TXT)

```

</details>

--- a/mobile/src/main/java/org/openhab/habdroid/background/ItemUpdateWorker.kt
+++ b/mobile/src/main/java/org/openhab/habdroid/background/ItemUpdateWorker.kt
@@ -14,7 +14,7 @@
 package org.openhab.habdroid.background
 
 import android.content.Context
-import android.content.pm.ServiceInfo.FOREGROUND_SERVICE_TYPE_DATA_SYNC
+import android.content.pm.ServiceInfo.FOREGROUND_SERVICE_TYPE_SHORT_SERVICE
 import android.os.Parcelable
 import android.util.Log
 import android.widget.Toast
@@ -266,7 +266,11 @@ class ItemUpdateWorker(context: Context, params: WorkerParameters) : CoroutineWo
             .addAction(R.drawable.ic_clear_grey_24dp, context.getString(android.R.string.cancel), cancelIntent)
             .build()
 
-        return ForegroundInfo(NOTIFICATION_ID_BACKGROUND_WORK_RUNNING, notification, FOREGROUND_SERVICE_TYPE_DATA_SYNC)
+        return ForegroundInfo(
+            NOTIFICATION_ID_BACKGROUND_WORK_RUNNING,
+            notification,
+            FOREGROUND_SERVICE_TYPE_SHORT_SERVICE
+        )
     }
 
     private fun buildOutputData(hasConnection: Boolean, httpStatus: Int, sentValue: String? = null): Data {

